src/payment/strategies/paypalCommerce/paypalCommerce-script-loader.ts

```diff
@@ -0,0 +1,27 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import { PaymentMethodClientUnavailableError } from '../../errors';
+
+import { PaypalCommerceHostWindow, PaypalCommerceSDK } from './paypalCommerce-sdk';
+
+export default class PaypalCommerceScriptLoader {
+    private _window: PaypalCommerceHostWindow;
+
+    constructor(
+        private _scriptLoader: ScriptLoader
+    ) {
+        this._window = window;
+    }
+
+    async loadPaypalCommerce(clientId: string = '', currency: string = 'USD'): Promise<PaypalCommerceSDK> {
```

Comment:
I think it might be better to not set a default value for both parameters. Things would not work properly without a client ID. And it's not correct to assume that the default currency is USD. If there are no default values to the method, the compiler can help us by warning us when we forget to pass one of these required arguments in.

Reply:
Thank you!
I agree with you. It will be better